conans/client/tools/oss.py

```diff
@@ -35,14 +35,32 @@ def cpu_count():
 def detected_architecture():
     # FIXME: Very weak check but not very common to run conan in other architectures
     machine = platform.machine()
-    if "64" in machine:
+    if "ppc64le" in machine:
+        return "ppc64le"
+    elif "ppc64" in machine:
+        return "ppc64"
+    elif "ppc" in machine:
+        return "ppc"
```

Comment:
``ppc`` doesn't seem to be an existing arch in ``settings.yml``

Reply:
so, remove it?

Reply:
Either remove it, (maybe printing some warning message that the arch is something, but doesn't match a known architecture), or add it to the default ``settings.yml``, whatever makes more sense.

Reply:
I'll just remove for now - it's very unlikely to be used in real life, and we can re-add on demand